### PR TITLE
Adding unit tests for com.alibaba.excel.util and com.alibaba.excel.metadata.

### PR DESCRIPTION
**Pull request by EdgarScott**
I've analysed your codebase and noticed that `com.alibaba.excel.util` and `com.alibaba.excel.metadata` are not fully tested.
I've written some tests for the methods in these classes with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for other classes, I would be more than happy to look further.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -77,7 +77,36 @@
...
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>1.6.5</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.10.19</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (paulkennethkent) [inline pom.xml]: powermock, and mockito deps are not needed

File: src/test/java/com/alibaba/easyexcel/test/util/StringUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+  }
+
+  @Test
+  public void sortStringArrayInput0Output0() {
+  	Assert.assertArrayEquals(new String[] {}, StringUtils.sortStringArray(new String[]{}));
+  }
+
+  @Test
+  public void sortStringArrayInput2Output2() {
+    final String[] array = {
```
Comment (paulkennethkent) [inline src/test/java/com/alibaba/easyexcel/test/util/StringUtilsTest.java:499]: If you could find out which phase this was generated in we can look into how we can stop this from happening

**Review by paulkennethkent — CHANGES_REQUESTED** [review]

File: src/test/java/com/alibaba/easyexcel/test/util/StringUtilsTest.java (hunk 2 above)
Comment (EdgarScott) [reply src/test/java/com/alibaba/easyexcel/test/util/StringUtilsTest.java:499]: Bugged: https://diffblue.atlassian.net/browse/TG-9317

File: pom.xml (hunk 1 above)
Comment (EdgarScott) [reply pom.xml]: removed

File: pom.xml [hunk 3]
```diff
@@ -77,7 +77,24 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
```
Comment (paulkennethkent) [inline pom.xml]: I don't think this dep is needed either

Comment (EdgarScott) [reply pom.xml]: Yep, didn't need deeptestutils either turns out. Removed both.

**Review by paulkennethkent — APPROVED** [review]

**Pull request closed without merging**